exercises/diffie-hellman/src/main/java/DiffieHellman.java

```diff
@@ -0,0 +1,20 @@
+import java.math.BigInteger;
```

Comment:
As per our [POLICIES](https://github.com/exercism/java/blob/master/POLICIES.md#adhere-to-best-practices) exercises with a difficulty of above 4 should not have starter implementations (except in extreme circumstances). As this has been rated as an 8 difficulty, we should remove this file and replace it with an empty `.keep`

Reply:
Removed ```DiffieHellman.java``` and replaced with ```.keep```.